pkg/cmd/target.go

```diff
@@ -739,6 +769,57 @@ func gardenWrapper(targetReader TargetReader, targetWriter TargetWriter, configR
 	return nil
 }
 
+func serverWrapper(reader ConfigReader, serverName string, kubeconfigReader KubeconfigReader) {
```

Comment:
I will make a suggestion for this function as it does not cover the cases I had in mind..

Reply:
Hi @neo-liang-sap, see PR for your fork https://github.com/neo-liang-sap/gardenctl/pull/1